### DISCUSSION TITLE
Elide 5 manual (link inside)

### DISCUSSION THREAD

**Discussion by Brutus5000** [yahoo/elide#1771] [Elide 5]
I know you guys have been working on Elide 5 for a very long time now, and I have also read a few buzzwords like OLTP like analytics. Unfortunately that doesn't really tell me much. Also Elide 5 (no longer?) show up in the changelog.
So I snooped around and actually figured out that I just need to increase the version number in the link.

https://elide.io/pages/guide/v5/01-start.html

Maybe you can add a drop-down to the version number in the header or similar?
Anyway I'm leaving the link here for other curious souls out there :)

**Comment by aklish** [comment]
Changes in Elide 5 can be found here:
https://elide.io/pages/guide/v5/17-migration.html

We haven't switched the docs over to Elide 5 yet since it is still in pre-release.  When we do, the main landing page will get updated, elide 5 docs will be the default getting started docs, and there will be a mechanism to switch to elide 4 documentation.

This project hasn't been announced yet, but if you want to see the sister project to elide (that corresponds to Elide 5 changes), you can check out [yavin.dev](https://yavin.dev).
